server/docker-compose.yaml

```diff
@@ -0,0 +1,54 @@
+version: '2'
+
+# starts 4 docker containers running minio server instances. Each
+# minio server's web interface will be accessible on the host at port
+# 9001 through 9004.
```

Comment:
can you please explain why we want 4? or more than 1?

Reply:
Unfortunately Minio requires a minimum of 4, its a known thing in Minio